### ISSUE TITLE
[Bug]: Main menu. DOM append in wrong sequence for user

### ISSUE THREAD

**Issue by vitalics** [magento-research/pwa-studio#633]
## This issue is for the following packages:

- [ ] `venia-concept`
- [ ] `pwa-buildpack`
- [ ] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`

## This issue is a:

- [ ] Bug
- [ ] Feature suggestion
- [ ] Documentation issue
- [ ] Other (Please Specify)

### Environment

| Question                    |    Answer     |
| --------------------------- | :-----------: |
| Magento version             |  |
| Operating System + version  |  |
| node.js version (`node -v`) |  |
| npm version (`npm -v`)      |  |

## Description
React render DOM for user in a wrong sequence: 
![image](https://user-images.githubusercontent.com/8816260/49739320-acdf0a80-fca2-11e8-9169-f0e8f3b5cbc0.png)
DOM. First element is Avatar with this image 
![image](https://user-images.githubusercontent.com/8816260/49739484-0ba48400-fca3-11e8-9ac6-dd50af2f9745.png). Next is User name and at last, render arrow 
![image](https://user-images.githubusercontent.com/8816260/49739538-2e369d00-fca3-11e8-94b0-1c835d0c7a6f.png)
DOM:
![image](https://user-images.githubusercontent.com/8816260/49739450-f9c2e100-fca2-11e8-8879-41610dbc62a8.png)

## Expected result:
Expected a render user Image, user name and arrow, as provided on mockups:
![image](https://user-images.githubusercontent.com/8816260/49739770-c03ea580-fca3-11e8-9c9c-919bcf584dcf.png)

## Possible solutions:
change render sequence

**Comment by codeAdrian** [comment]
Hello. I believe that this issue was related to the checkout layout being broken. The fix for it was pulled into release. Please check again and let us know if it persists. I have tried to reproduce this issue with fix applied and everything was looking good.

**Comment by ericerway** [comment]
Agreed. @vitalics can you please verify after yesterday's changes? Otherwise we can mark this as closed. Thanks.
